platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/lifecycle/info/NodeInfo.java

```diff
@@ -65,7 +65,7 @@ default boolean zeroStake() {
      * The stake weight of this node.
      * @return the stake weight
```

Comment:
is this consensus weight or stake value?

Reply:
Should be weight.